components/opponent/commons/opponent.lua

```diff
@@ -251,6 +251,9 @@ function Opponent.resolve(opponent, date, options)
 		for _, player in ipairs(opponent.players) do
 			if options.syncPlayer then
 				PlayerExt.syncPlayer(player, {savePageVar = not Opponent.playerIsTbd(player)})
+				if not player.team then
+					player.team = PlayerExt.syncTeam(player.pageName:gsub(' ', '_'), nil)
```

Comment:
why the `, nil`?
isn't that redundant?

Reply:
updated with nil removed, but nil check isnt needed in this case? (Or because the gsub already did it in place?)

Reply:
> why the `, nil`? isn't that redundant?

I now found why nil was there

This happens on prize pool when I removed nil 
![image](https://user-images.githubusercontent.com/88981446/224390652-f9446bef-1ad0-49da-bfe0-280e7f5ae735.png)

I've converted back to draft for now unless I need to put nil back

Reply:
why is the dev module on a wiki?
deleted it and am testing it via commons ...